### PR TITLE
Add 1.20 client RPM pipeline

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.20.groovy [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            post {
+                always {
+                    deleteDir()
+                }
+            }
+        }
+    }
+}
+
+void push_rpms(version, distro) {
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.20.groovy]: should these two be in a `lib` somewhere?

**Comment by ehelms** [comment]
@evgeni Attempted DRY, let me know what ya think

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/repoclosure.groovy [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/repoclosure.groovy]: this file is empty

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.20.groovy [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Mash Koji Repositories') {
+            agent { label 'sshkey' }
+
+            steps {
+
+                mash("foreman-client-mash-split-1.20.py")
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.20.groovy]: Could we put `1.20` in a variable?

**Pull request merged**